website/docs/kit/kit.module.css

```
@@ -0,0 +1,60 @@
+/* Section standfirst — sits directly under an h2. */
+.lead {
+  margin: 0 0 0.5rem;
+  font-size: var(--et-text-lg);
+  line-height: 1.6;
+  color: var(--et-text-secondary);
+}
+
+/* Tables on this page sit still — no row hover tint.
+   `.markdown` is an ancestor of `.page`, so qualifying with it outranks the
+   site-wide `.markdown table tbody tr:hover` rule without `!important`. */
+:global(.markdown) .page table tbody tr,
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)